spark/src/test/scala/org/apache/comet/CometFuzzTestSuite.scala

```diff
@@ -161,6 +162,18 @@ class CometFuzzTestSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("shuffle") {
+    val df = spark.read.parquet(filename)
```

Comment:
Does the data have complex type?

Reply:
Yes, the data has arrays and structs but not maps yet